src/components/EmailInput.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { TextField } from "@mui/material";
+import { FieldErrors, UseFormRegister } from "react-hook-form";
+
+import { LogInInputs } from "../types/LoginInput";
+
+interface EmailInputProps {
+  register: UseFormRegister<LogInInputs>;
+  errors: FieldErrors<LogInInputs>;
+}
+
+const EmailInput: React.FC<EmailInputProps> = ({ register, errors }) => {
```

Comment:
Instead of passing down the props explicitly here, it would be a lot nicer to use the form context and controller. Again, with this simple form this is also a good and simple solution.